lib/Order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu.rb'
+
+# This new class is going to create an order
+class Order
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+    @menu = Menu.new
+    @complete = false
+  end
+
+  def add_dish(name, amount = 1)
+    @name = name
+    @amount = amount
+    fail "This dish isn't available, please run menu.list_dishes to see the dishes" unless available
+
+    @dishes << { :name => name, :amount => amount }
+  end
+
+  def available
```

Comment:
The convention here would be for the method to end with a `?` to indicate that it returns a bool.  It also reads like a question, which is nice, don't you think?

Reply:
Yeh that would have been more applicable, will use that next time